engines/pyfunc-ensembler-service/pyfunc_ensembler_runner/__main__.py

```diff
@@ -28,14 +29,14 @@
     dest="log_level",
     choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
     help="Set the logging level",
-    default=logging.DEBUG,
+    default=logging.WARNING,
 )
```

Comment:
just remove this `--log-level ` flag and set the log level via env var only (`WARNING` as the default value)

Reply:
also, usually the command flag has higher priority than the ENV var. if we replace the flag with ENV, it will create confusion to our users.